region/lowernorfair/east/Main Hall.json

```
@@ -410,39 +382,73 @@
       ]
     },
     {
-      "id": 23,
-      "link": [3, 3],
-      "name": "Crystal Flash",
+      "id": 29,
+      "link": [4, 4],
+      "name": "Come Down",
```

```suggestion
      "name": "Base (Come Down)",
```

By starting the strat name with "Base", the randomizer will omit these strat names from the spoiler routes. I think it also just makes sense since this is the basic way of entering the room.

---

I thought it had to have the exact name as `Base`. I was fighting to see which strats to have with that name. I think there are a lot of strats that should have their name changed to clean up the spoiler logs.

This does feel weird and hacky though, just checking the start of the strat name, I wonder if it should be a separate field?